.github/new_pull_request.md

```diff
@@ -0,0 +1,16 @@
+### Problem Statement
+
+
+### Solution
+
+
+### PRT test Cases
+
+
+### Checklist
+- [ ] I have tested the changes locally.
+- [ ] CherryPick labels added
+
```

Comment:
I'm not sure we need a check list. The actions will fail if you don't have the cherry-pick label and not all tests need to be ran locally. We mostly care that the test passes in CI.

Reply:
Okay, removing this section.